tasks/build.rake

```diff
@@ -80,3 +80,12 @@ task 'build:aws-sdk-ssooidc' do
     spec_writer.write_files(generator.spec_files)
   end
 end
+
+task 'gem-name', [:service_id] do |_, args|
```

Comment:
I'd rather a task that returns the Service codegen object. Must this take service id as an argument? Wondering if service id can be a property of Service and this task can somehow take a different argument

Reply:
Oh.. I guess this cannot return an object.